exercises/concept/elyses-enchantments/.docs/introduction.md

```diff
@@ -1,12 +1,84 @@
 # Introduction
 
 In Javascript, an array is a list-like structure with no fixed length which can hold any type of primitives or objects, even mixed types.
-The array elements can be accessed by their index.
 
-Example of an array declaration and accessing an element by index:
+To create an array, add elements between square brackets `[]`. To read from the array, put the index in square brackets `[]` after the identifier. The indices of an array start at zero.
```

Comment:
General comment: please copy this content into the introduction.md file of the arrays concept. Also make sure everything that is mentioned here is also included in the about.md file of that concept (but don't remove the additional advanced content that is in there currently, you want to merge the two so to speak).

Reply:
should I copy over the whole file or just overwrite what is already in that file?

Reply:
Overwrite all content in `concepts/arrays/introduction.md` with the content of the exercise introduction.md file you created here. For the about.md file (`concepts/arrays/about.md`) you need to manually merge the content of the two files though so that the current information that is unique to the about.md file is still present afterwards.